lib/maths/CBoostedTreeLoss.cc

```diff
@@ -4,6 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+#include "maths/CBoostedTreeUtils.h"
```

Comment:
```suggestion
#include <maths/CBoostedTreeUtils.h>
```

Reply:
Done in [c83ffe6](https://github.com/elastic/ml-cpp/pull/1901/commits/c83ffe6f36344efce60c1b32d68f0cb671b2e48a).